cmd/argocd/commands/app_actions.go

```diff
@@ -156,10 +150,23 @@ func NewApplicationResourceActionsRunCommand(clientOpts *argocdclient.ClientOpti
 				ResourceName: objResourceName,
 				Group:        gvk.Group,
 				Kind:         gvk.Kind,
-				Action:       actionName,
+				Action:       actionNameOnly,
 			})
 			errors.CheckError(err)
 		}
 	}
 	return command
 }
+
+func parseActionName(action string) (string, string, string) {
+	actionNameSplit := strings.Split(action, ":")
+	if len(actionNameSplit) != 2 {
+		log.Fatal("Action name is malformed")
+	}
+	actionName := actionNameSplit[1]
+	groupKindSplit := strings.Split(actionNameSplit[0], "/")
+	if len(groupKindSplit) != 2 {
+		log.Fatal("Action name is malformed")
+	}
+	return groupKindSplit[0], groupKindSplit[1], actionName
```

Comment:
It appears we are making breaking changes here by requiring a new convention for running actions. I think it's okay if we start introducing a new convention, but we may need to preserve previous flags for backwards compatibility since many pipelines are depending on this behavior.

Reply:
Introduced backwards compatibility with the following message:

```
$ argocd app actions run blue-green resume --kind Rollout

Warning: "resume" action has been deprecated. Please run the action as

	argocd app run blue-green argoproj.io/Rollout/resume

```